sfa_dash/static/js/report-form-utilities.js

```diff
@@ -152,14 +152,17 @@ report_utils.toggle_reference_dependent_metrics = function(){
 current_units = null;
 
 
-report_utils.unset_units = function(){
+report_utils.unset_units = function(filter_callback=null){
```

Comment:
what's the purpose of `=null` here and below?

Reply:
To make the filtering optional argument. Right now, it's always called with a callback. Due to recently kicking around the idea of merging all three types of report forms I thought this should be an optional argument because I don't know how that'll all tie together.